src/clj/fluree/db/query/exec/eval.cljc

```diff
@@ -382,6 +382,7 @@
        :cljs (js/Date.))
     const/iri-xsd-dateTime))
 
+;; TODO - date functions below all look incorrect for CLJS - should (string? datetime) be (map? datetime)?
```

Comment:
These have all been fixed here: https://github.com/fluree/db/pull/915

I need to cherry pick those out into a separate PR.